doc/source/tokudb/tokudb_faq.rst

```diff
@@ -0,0 +1,244 @@
+.. _tokudb_faq:
+
+==========================
+Frequently Asked Questions
+==========================
+
+This section contains frequently asked questions regarding |TokuDB| and related software. 
+
+.. contents::
+   :local:
+   :depth: 1
+
+Transactional Operations
+------------------------
+
+**What transactional operations does TokuDB support?**
+
+|TokuDB| supports ``BEGIN TRANSACTION``, ``END TRANSACTION``, ``COMMIT``, ``ROLLBACK``, ``SAVEPOINT``, and ``RELEASE SAVEPOINT``.
+
+TokuDB and the File System
+--------------------------
+
+**How can I determine which files belong to the various tables and indexes in my schemas?**
+
+The :table:`tokudb_file_map` plugin lists all Fractal Tree Indexes and their corresponding data files. The ``internal_file_name`` is the actual file name (in the data folder).
+
+.. code-block:: sql
+
+ mysql> SELECT * FROM information_schema.tokudb_file_map;
+
+ +--------------------------+---------------------------------------+---------------+-------------+------------------------+
+ | dictionary_name          | internal_file_name                    | table_schema  | table_name  | table_dictionary_name  |
+ +--------------------------+---------------------------------------+---------------+-------------+------------------------+
+ | ./test/tmc-key-idx_col2  | ./_test_tmc_key_idx_col2_a_14.tokudb  | test          | tmc         | key_idx_col2           |
+ | ./test/tmc-main          | ./_test_tmc_main_9_14.tokudb          | test          | tmc         | main                   |
+ | ./test/tmc-status        | ./_test_tmc_status_8_14.tokudb        | test          | tmc         | status                 |
+ +--------------------------+---------------------------------------+---------------+-------------+------------------------+
+
+.. _tokudb_full_disks:
+
+Full Disks
+----------
+
+**What happens when the disk system fills up?**
+
+The disk system may fill up during bulk load operations, such as ``LOAD DATA IN FILE`` or ``CREATE INDEX``, or during incremental operations like ``INSERT``.
+
+In the bulk case, running out of disk space will cause the statement to fail with ``ERROR 1030 (HY000): Got error 1 from storage engine``. The temporary space used by the bulk loader will be released. If this happens, you can use a separate physical disk for the temporary files (for more information, see :variable:`tokudb_tmp_dir`).
```

Comment:
See https://tokutek.atlassian.net/browse/DB-740 that we are trying to avoid fixing anytime soon, maybe update the docs to state that on ENOSPC in tmp TokuDB will assert the server to prevent data corruption to existing data files.